src/os_unix.c

```diff
@@ -2259,7 +2259,10 @@ vim_is_xterm(char_u *name)
 use_xterm_like_mouse(char_u *name)
 {
     return (name != NULL
-	    && (term_is_xterm || STRNICMP(name, "screen", 6) == 0));
+	    && (term_is_xterm || STRNICMP(name, "screen", 6) == 0
+		|| STRNICMP(name, "st", 3) == 0 
```

Comment:
To check if `name` is exactly the same as `"st"`, using `STRICMP()` is better, I think.
When I saw this code, at first I thought "3" was the mistake of "2".